recipe/meta.yaml

```diff
@@ -15,14 +15,15 @@ source:
 
 build:
   number: {{ build }}
-  string: "{{processor}}_h{{ PKG_HASH }}_{{ build }}"
+  skip: True  # [win or py2k]
+  string: "{{ processor }}_h{{ PKG_HASH }}_{{ build }}"
+
+requirements:
+  host:
+    - python
```

Comment:
In development I found having *any* requirements in the main package setup caused problems in the sub-packages. Possibly we should include the skip command here, but including the host requirements I think will cause trouble later.

Reply:
This is surprising because the metapackage I came across use this pattern, see for example, https://github.com/conda-forge/matplotlib-feedstock, https://github.com/conda-forge/hdf5-feedstock, https://github.com/conda-forge/hyperspy-feedstock.

Reply:
It's possible that some combinations of adding requirements to the main package and the subpackages would work - I didn't try all combinations of options. But I found it a lot easier to control the build system when I just included the requirements in the subpackages, and in this case I suspect that we'd end up with the GUI and CLI builds having requirements for python when they don't need it.